.projenrc.ts

```diff
@@ -49,9 +49,4 @@ project.package.addField('jsiiRosetta', {
 
 project.preCompileTask.exec('./build-lambda.sh');
 
-project.package.addField('resolutions', {
-  'trim-newlines': '3.0.1',
-  'xmldom': 'github:xmldom/xmldom#0.7.0', // TODO: remove this when xmldom^0.7.0 is released in npm
-});
-
```

Comment:
`xmldom` was not used anymore

`trim-newlines` is a dev dep and already was at that version